notebooks/stable-fast-3d/stable-fast-3d.ipynb

```diff
@@ -0,0 +1,347 @@
+{
```

Comment:
<p>please add possibility to view conversion code as comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2388/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>added</p>